app/sidekiq/evss/disability_compensation_form/submit_form526.rb

```diff
@@ -116,8 +116,14 @@ def perform(submission_id)
                          service = BenefitsClaims::Service.new(icn)
                          raw_response = service.submit526(body)
                          # 4. convert LH raw response to a FormSubmitResponse for further processing (claim_id, status)
```

Comment:
@aurora-a-k-a-lightning Does @department-of-veterans-affairs/Disability-Experience need to replace the following CODEOWNERS for lib/lighthouse/benefit_claims:

lib/lighthouse/benefit_claims @department-of-veterans-affairs/benefits-management-tools-be @department-of-veterans-affairs/va-api-engineers @department-of-veterans-affairs/backend-review-group

Reply:
Same for spec/lib/lighthouse/benefits_claims:
spec/lib/lighthouse/benefits_claims @department-of-veterans-affairs/benefits-management-tools-be @department-of-veterans-affairs/va-api-engineers @department-of-veterans-affairs/backend-review-group

Reply:
@RachalCassity  sure! i can look at this 🧐

Reply:
@RachalCassity this has been updated. Is this what you envisioned?

Reply:
My team should still be CODEOWNERS on the files mentioned above @RachalCassity

Reply:
@jerekshoe it is done! please check again to confirm 😄

Reply:
@RachalCassity @jerekshoe updated. please confirm this is what we want :)